catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/pom.xml

```diff
@@ -235,7 +235,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.55</minimum>
+                                            <minimum>0.52</minimum>
```

Comment:
❓ What was it reporting? Why not `0.50`?

Reply:
reduced to 0.50